src/shader/cel.rs

```diff
@@ -45,13 +45,13 @@ impl<'a> Pipeline for CelShader<'a> {
         let v_index = *v_index as usize;
         // Find vertex position
         let v_pos = Vec4::from_point(self.mesh.position(v_index));
+        let v_pos_cam_4 = (self.mvp * v_pos).homogenized();
         // Calculate vertex position in camera space
-        let v_pos_cam = Vec3::from(self.mvp * v_pos).into_array();
+        let mut v_pos_cam = Vec3::from(v_pos_cam_4).into_array();
```

Comment:
I think you can do this without the separate variable now. (Your changes to this shader should look the same as your changes to outline.rs)